int32 serialID

int32 beltDirection

int32 speed

int32 stop
